.github/workflows/ci.yml

```diff
@@ -104,6 +106,8 @@ jobs:
       - name: cargo test --release --workspace
         run: |
           export RUSTFLAGS="-D warnings"
+          # add homebrew library path for z3
+          export RUSTFLAGS="$RUSTFLAGS -Clink-arg=-L/opt/homebrew/lib -Clink-arg=-Wl,-rpath,/opt/homebrew/lib"
```

Comment:
Does this need to be in CI?  Can this be done in a `build.rs` or something?

Reply:
@fw-immunant Should I add the comment, since I just pushed a new rebase?